src/LuaEngine.cpp

```diff
@@ -1123,6 +1138,79 @@ static int l_engine_set_sector_map_automatic_system_selection(lua_State *l)
 	return 0;
 }
 
+static int l_engine_sector_map_get_route(lua_State *l) {
+	SectorView *sv = Pi::game->GetSectorView();
+	std::vector<SystemPath> route = sv->GetRoute();
+
+	lua_newtable(l);
```

Comment:
Not sure if this works here, but LuaTable might be a better fit (no low-level lua functions)

Reply:
To be honest, this is the first time I've worked with Lua at all..so I stumbled through moving data back and forth. If there's something already written for doing that, I'm open to it.